libs/financial-aid/shared/src/lib/const.ts

```diff
@@ -11,3 +11,5 @@ export const AllowedFakeUsers = [
   '0000000002', // Veita user
   '0000000003', // User with an application which needs data
 ]
+
+export const LoginStatusPage = '/stada'
```

Comment:
So what I was thinking here is something like this: 

```
export const Routes = {
  status: '/stada',
  statusFileUpload: (id: string) => `/stada/${id}/gogn`,
}
```

Etc....

Then we would never have hardcoded values in our components but would only refer to the `Routes` obj. Does this make sense? It could also be a separate PR.